client/src/components/AudienceViews/Waiting.vue

```diff
@@ -1,3 +1,41 @@
 <template>
-  <div>Audience Waiting</div>
-</template>
\ No newline at end of file
+  <div class="background-matte center">
+    <div class="txt-container center px-5 pt-3">
+      <h1>The Room You Joined is Full :(</h1>
+      <p class="mt-2">
+        Due to all player slots being filled, we 
```

Comment:
What is this styling?

Reply:
mt-2 is what I am assuming you are referring to. It is a helper class in vuetify that shortens common css that is written. mt-2 is margin-top 2 (2 being a special unit they use in their framework)

Reply:
No, I was talking about the text. Why is there a breaking in that specific spot?

Reply:
readability

Reply:
See other comment chain comment regarding styling.